perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -1236,10 +1237,12 @@ sub construct_bin_request_form {
         for my $container (
             @{ $c->stash->{property}{containers_for_delivery} } )
         {
+            my $id;
```

Comment:
Doesn't matter, but can't see why ID was pulled out of the if here, it's never used after.